tests/readrows_test.go

```diff
@@ -324,3 +324,24 @@ func TestReadRows_Retry_StreamReset(t *testing.T) {
 	assert.True(t, cmp.Equal(loggedRetry.req.GetRows().GetRowRanges()[0].StartKey, &btpb.RowRange_StartKeyOpen{StartKeyOpen: []byte("abar")}))
 }
 
+// TestReadRows_Generic_EmptyTableNoRows tests that reads on an empty table returns 0 rows.
+func TestReadRows_Generic_EmptyTableNoRows(t *testing.T) {
+	// 1. Instantiate the mock server
+	recorder := make(chan *readRowsReqRecord, 3)
+	sequence := []*readRowsAction{
```

Comment:
As there is only one action, you can make it simpler with:
`action := &readRowsAction{chunks: []chunkData{}}`

And call `mockReadRowsFnSimple(recorder, action)`

Reply:
Done.